mcp-server/main.py

```
@@ -1,17 +1,9 @@
-from datetime import datetime, UTC
+from fastmcp import FastMCP, Context
 import jwt
-from loguru import logger
 import os
+from loguru import logger
 from dotenv import load_dotenv
-from contextvars import ContextVar
-from functools import wraps
-import time
-import json
+from typing import Dict, Any, Optional, List
```

Ruff (0.11.9)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)